### PR TITLE
Token

### PR DESCRIPTION
**Pull request by c9dong**

### CODE REVIEW

File: JWT/jwtUtil.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var jwt = require('jsonwebtoken');
+
+var secret = process.env.JWT_SECRET;
+var algorithm = 'RS256';
```
Comment (clemmy) [inline JWT/jwtUtil.js]: @jasonf7

File: JWT/jwt.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var jwtUtil = require('./jwtUtil');
+
+function* jwtChecker(next) {
+  if (this.path != "/shared/auth") {
+    var token = this.headers.token;
```
Comment (clemmy) [inline JWT/jwt.js]: this.headers.Authorization;

var split = token.split('Bearer ');
if (split.length == 2) {
  var id = jwtUtil.decode(split[1]);
} else {
  // error
}

Also include 'Bearer ' before the token on the front end.

**Review by clemmy — CHANGES_REQUESTED** [review]

File: JWT/jwtUtil.js (hunk 1 above)
Comment (jasonf7) [reply JWT/jwtUtil.js]: You know what to do david

Comment (clemmy) [reply JWT/jwtUtil.js]: hello?

**Review by clemmy — APPROVED** [review]

**Comment by clemmy** [comment]
`claps`

**Pull request merged**